index.html

```diff
@@ -1,49 +1,106 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
-    <meta charset="utf-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="stylesheet" href="pages/index.css">
-    <title>Pink App</title>
-  </head>
-  <body class="page">
-    <div class="page__container">
-      <section class="research">
-        <h2 class="visually-hidden">Исcледование приложения</h2>
-        <picture>
-          <source srcset="images/research-timer-tablet.png" media="(max-width: 660px)">
-          <img src="images/research-timer-desktop.png" alt="Картинка таймера" class="research__image">
-        </picture>
-        <div class="research__inner-wrapper">
-          <p class="research__title">Эффект на&nbsp;24&nbsp;часа!</p>
-          <p class="research__paragraph">
-            Британские ученые провели исследования и доказали, что положительный
-            эффект от использования приложения длится целые сутки!
-          </p>
-          <a href="#" target="_blank" class="research__link">Узнать больше об исследовании</a>
+
+<head>
+  <meta charset="utf-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="pages/index.css">
+  <title>Pink App</title>
+</head>
+<body class="page">
+  <div class="page__container">
+    
+    <section class="research">
+      <h2 class="visually-hidden">Исcледование приложения</h2>
+      <picture>
+        <source srcset="images/research-timer-tablet.png" media="(max-width: 660px)">
+        <img src="images/research-timer-desktop.png" alt="Картинка таймера" class="research__image">
+      </picture>
+      <div class="research__inner-wrapper">
+        <p class="research__title">Эффект на&nbsp;24&nbsp;часа!</p>
+        <p class="research__paragraph">
+          Британские ученые провели исследования и доказали, что положительный
+          эффект от использования приложения длится целые сутки!
+        </p>
+        <a href="#" target="_blank" class="research__link">Узнать больше об исследовании</a>
+      </div>
+    </section>
+    <section class="reviews">
+      <svg xmlns="http://www.w3.org/2000/svg" class="reviews__quotes" width="79" height="63" viewBox="0 0 79 63"
+        fill="none">
+        <path
+          d="M35.1468 6.97257L30.8388 0C28.3886 1.48142 0 12.0588 0 39.1293C0 46.2698 0.341426 63 18.9793 63C30.2162 63 32.9778 52.8473 32.9778 49.5783C32.9778 27.8508 11.5081 35.5838 11.5081 35.5838C11.5081 17.7771 31.3911 9.06631 35.1468 6.97257Z"
+          fill="#EFEFEF" />
+        <path
+          d="M79 6.97257L74.7322 0C72.302 1.48142 43.8833 12.0588 43.8833 39.1293C43.8833 46.2698 44.2247 63 62.8726 63C74.0995 63 76.8611 52.8473 76.8611 49.5783C76.8611 27.8508 55.3914 35.5838 55.3914 35.5838C55.3914 17.7771 75.3146 9.06631 79 6.97257Z"
+          fill="#EFEFEF" />
+      </svg>
+      <div class="reviews__contents">
+        <button class="reviews__button" aria-label="previous">
```

Comment:
@Woolfs1423 Thanks for following my advice and improving elements as suggested!

Now let's fix a small BEM mistake here. I see that you tried to create class names to style differences between left and right arrows. In our case, we want to utilize BEM modifiers. Considering the potential for multiple options such as `right`, `left`, and theoretically `up` and `down`, we should opt for a **key-value type** on the modifier. I suggest using the word `direction` as a key and `right` and `left` as values. Please keep in mind that modifiers are separated from blocks or elements by a single underscore `_`.

Now, let's take a look at what we've got. 

```CSS
/* slider__arrow-button.css */
.slider__arrow-button {
    border: none;
    background: none;
}

@media screen and (max-width:1199px) {
    .slider__arrow-button {
        display: none;
    }
}

/* slider__arrow-button_direction_left.css */
.slider__arrow-button_direction_left {
    position: absolute; /* present on both left and right arrow buttons */
    top: 140px; /* present on both left and right arrow buttons */
    right: 440px;
}

/* slider__arrow-button_direction_right.css */
.slider__arrow-button_direction_right {
    position: absolute; /* present on both left and right arrow buttons */
    top: 140px; /* present on both left and right arrow buttons */
    left: 440px;
}
```

Do you see how we got some identical styles set for both `slider__arrow-button_direction_left` and `slider__arrow-button_direction_right`? According to BEM principles, we should aim to avoid code duplication and only style differences within BEM modifiers. Styles that are the same should be defined on the base class, in this case, `slider__arrow-button`.

To rectify this, let's move `position: absolute;` and `top: 140px;` to the `slider__arrow-button.css` file and remove those lines from the modifier files.

Lastly, let's make sure our files and folder structure follows [BEM Nested](https://github.com/p-foundation/pink-app/blob/main/blocks/README.md#key-value-modifiers). You should end up with a structure like this:
```bash
blocks/
  # other blocks
  slider/                               
    slider.css
    __arrow-button/
      _direction/
        slider__arrow-button_direction_left.css
        slider__arrow-button_direction_right.css
   # other elements
```